packages/calypso-config/src/desktop.ts

```diff
@@ -20,6 +20,7 @@ const features = {
 	// that may/may not be relevant to override for the Desktop app.
 	'signup/social': false,
 	'login/magic-login': false,
+	'login/qr-code-login': false,
```

Comment:
Correct me if I'm wrong but I think this would only be necessary if this feature flag was enabled in production.

Reply:
I am not sure I just added it as a safe measure.